packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -532,25 +532,18 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         .domain(this._isRtl ? [xMax, xMin] : [xMin, xMax])
         .nice()
         .range([
-          this.margins.left! + this._domainMargin,
-          containerWidth - this.margins.right! - this._barWidth - this._domainMargin,
+          this.margins.left! + this._domainMargin - this._barWidth / 2,
+          containerWidth - this.margins.right! - this._barWidth / 2 - this._domainMargin,
         ]);
     } else if (this._xAxisType === XAxisTypes.DateAxis) {
       const sDate = d3Min(this._points, (point: IVerticalBarChartDataPoint) => point.x as Date)!;
       const lDate = d3Max(this._points, (point: IVerticalBarChartDataPoint) => point.x as Date)!;
       xBarScale = d3ScaleUtc()
-        .domain([sDate, lDate])
-        .range(
-          this._isRtl
-            ? [
-                containerWidth - this.margins.right! - this._barWidth / 2 - this._domainMargin,
-                this.margins.left! + this._domainMargin + this._barWidth / 2,
-              ]
-            : [
-                this.margins.left! + this._domainMargin + this._barWidth / 2,
-                containerWidth - this.margins.right! - this._barWidth / 2 - this._domainMargin,
-              ],
-        );
+        .domain(this._isRtl ? [lDate, sDate] : [sDate, lDate])
```

Comment:
for RTL domain is the same but range is reversed.

Reply:
Either way is fine. I modified this to make the changes look similar to the numeric scale.

Reply:
reverted the change